atc/atccmd/command.go

```diff
@@ -119,8 +119,9 @@ type RunCommand struct {
 
 	Postgres flag.PostgresConfig `group:"PostgreSQL Configuration" namespace:"postgres"`
 
-	APIMaxOpenConnections     int `long:"api-max-conns" description:"The maximum number of open connections for the api connection pool." default:"10"`
-	BackendMaxOpenConnections int `long:"backend-max-conns" description:"The maximum number of open connections for the backend connection pool." default:"50"`
+	ConcurrentRequestLimits   map[wrappa.LimitedRoute]int `long:"concurrent-request-limit" description:"Limit the number of concurrent requests to an API endpoint (Example: ListAllJobs:5)"`
```

Comment:
Since it's a map, it can contain multiple limits, hence plural. However the way you do this is by passing the flag multiple times, so the long flag name is singular. This approach is consistent with [worker bundled resource types](https://github.com/concourse/concourse/blob/master/atc/atccmd/command.go#L159) and [extra metrics tags](https://github.com/concourse/concourse/blob/master/atc/atccmd/command.go#L164).

Reply:
@pivotal-jamie-klassen Ah I see, that makes sense!